src/test/sql/sqlite_testrunner/sqlite_wrapper.cpp

```diff
@@ -92,7 +92,8 @@ std::shared_ptr<Table> SQLiteWrapper::execute_query(const std::string& sql_query
   std::vector<std::string> queries;
   boost::algorithm::split(queries, sql_query, boost::is_any_of(";"));
 
-  queries.erase(std::remove_if(queries.begin(), queries.end(), [](std::string const& query) { return query.empty(); }),
+  queries.erase(std::remove_if(queries.begin(), queries.end(),
+                               [](std::string const& query) { return query.empty() || query == "\n"; }),
```

Comment:
Dumb question: if there are multiple newlines after a query, this wouldn't fix it, right? Because two newlines is what I had at the end of TPC-H 15 after I removed the `DROP VIEW` statement.

Does it maybe make sense to use the SQLPipeline's functionality of splitting up a query into statements here?

Reply:
Right, overlooked that.

I'm not sure. I mean we could, but that would mean parsing the sql strings with our hsql::SQLParser etc, which we don't need for the SQLiteWrapper. I will check it out and see how it looks in the end.